src/components/media-player/ha-media-player-browse.ts

```diff
@@ -92,11 +94,54 @@ export class HaMediaPlayerBrowse extends LitElement {
     this._navigate(item);
   }
 
+  private _renderError(err: { message: string; code: string }) {
+    if (err.message === "Path does not exist.") {
+      return html`
+        <h2>No local media found.</h2>
+        <p>
+          It looks like you have not yet created a media directory.
+          <br />Create a directory with the name <b>"media"</b> in the
+          configuration directory of Home Assistant (${this.hass.config.config_dir}). <br />Place your video,
+          audio and image files in this directory to be able to browse and play
+          them in the browser or on supported media players.
+        </p>
+
+        <p>
+          Check the
+          <a
+            href="https://www.home-assistant.io/integrations/media_source/#local-media"
```

Comment:
Should we use the RC site if the version contains a "b" here?
This will go to a non-exsisting site during the beta

Reply:
If we do that, we should get a helper method and apply that everywhere. I think it's fine for now.